api/swagger.yaml

```diff
@@ -6459,6 +6459,11 @@ paths:
           description: "Target build stage"
           type: "string"
           default: ""
+        - name: "outputs"
+          in: "query"
+          description: "BuildKit output configuration"
+          type: "string"
```

Comment:
IIUC, although correct that this is a string, it is a JSON configuration encoded as string; should we specify the format here? (or at least mention it's JSON)?

Also, can you add this to the API changes doc? https://github.com/moby/moby/blob/master/docs/api/version-history.md#v140-api-changes

Reply:
About half of the fields of this struct are actually JSON (eg. buildargs) but they all seem to be defined as string here.

Reply:
Added to `version-history.md`